src/chunk.js

```diff
@@ -43,18 +44,37 @@ const resolved = Promise.resolve();
  * there is time.
  *
  * @param {!Node|!./service/ampdoc-impl.AmpDoc} nodeOrAmpDoc
- * @param {function()} fn Function that will be called as a "chunk".
+ * @param {function(?IdleDeadline)} fn
  */
-export function chunk(nodeOrAmpDoc, fn) {
+export function startupChunk(nodeOrAmpDoc, fn) {
```

Comment:
Chunking used to only be used for startup tasks, so I guess the `nochunking=1` test query was very useful for testing. There's less of a need for generic background task chunks.

Added it below though since it's better to be consistent here.